src/main/kotlin/org/stellar/walletsdk/utils/Transaction.kt

```diff
@@ -0,0 +1,36 @@
+package org.stellar.walletsdk.utils
+
+import org.stellar.sdk.Network
+import org.stellar.sdk.Operation
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+import org.stellar.sdk.responses.AccountResponse
+
+class Transaction {
+  companion object {
+    fun buildTransaction(
+      sourceAddress: String,
+      server: Server,
+      network: Network,
+      operations: List<Operation>
+    ): Transaction {
+      val sourceAccount: AccountResponse?
+
+      try {
+        sourceAccount = server.accounts().account(sourceAddress)
+      } catch (e: Error) {
+        throw Error("Source account was not found")
+      }
+
+      // TODO: add memo
+      // TODO: update max fee
+      // TODO: add time bounds
```

Comment:
I don't want to spend too much time on these now, but we will need to add them at some point.